modules/client/src/main/java/org/apache/ignite/client/IgniteClient.java

```diff
@@ -88,6 +89,8 @@ class Builder {
         /** Retry policy. */
         private RetryPolicy retryPolicy = new RetryReadPolicy();
 
+        private LoggerFactory loggerFactory;
```

Comment:
javadoc

Reply:
afaik, javdocs on a private members are not mandatory in AI3